examples/wallet/script/src/bin/execute.rs

```diff
@@ -116,3 +115,43 @@ fn main() {
   assert_eq!(charlie_balance, 10);
   assert_eq!(new_alice_balance, alice_balance - 10);
 }
+
+#[cfg(test)]
+mod tests {
+  use athena_interface::{Address, HostDynamicContext, HostStaticContext, MockHost, ADDRESS_ALICE};
+  use athena_sdk::{AthenaStdin, ExecutionClient};
+  use athena_vm_sdk::Pubkey;
+
+  #[test]
+  fn deploying_other_contract() {
+    athena_sdk::utils::setup_logger();
+
+    let mut host = MockHost::new_with_context(
+      HostStaticContext::new(ADDRESS_ALICE, 0, ADDRESS_ALICE),
+      HostDynamicContext::new([0u8; 24], ADDRESS_ALICE),
+    );
+    let address = super::spawn(&mut host, Pubkey::default()).unwrap();
+
+    // deploy other contract
+    let code = b"some bad really code".to_vec();
```

Comment:
```suggestion
    let code = b"some really bad code".to_vec();
```

this made me laugh :)